packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -294,6 +294,7 @@ const processPasswordElement = (node: HTMLInputElement, registry: containerDeskt
                   <input
                     type="text"
                     placeholder="Username"
+                    aria-label="Username"
```

Comment:
We could also probably use `role="cell"` on a proper level - probably on the div that is the most outer for given cell.

Reply:
This also counts for other cells we need to operate over.